topic04-git-introduction/book-git-intro/01.Terminal.md

```diff
@@ -0,0 +1,51 @@
+# Terminal
+
+## What is a terminal? 
+The terminal is an interface in which you can type and execute text-based commands.
+
+![A terminal view](../images/terminal.png)
```

Comment:
for some reason this image is not rendering locally when I run the site on my machine - is it rendering ok for others?  (same issue for other images in this file & Git-and-github.md below)